### DISCUSSION TITLE
Explorer plugin / Everything / macros

### DISCUSSION THREAD

**Discussion by AlanEdd** [Flow-Launcher/Flow.Launcher#2866] [Q&A] [answered]
Hello there,
I was reading the documentation and I see a link to using Everything within Flow Launcher.
https://www.voidtools.com/support/everything/searching/

Am I right in assuming the operators and wildcards are supported but not the macros? (They don't seem to work for me anyway)

audio:	Search for audio files.
zip:	Search for compressed files.
doc:	Search for document files.
exe:	Search for executable files.
pic:	Search for picture files.
video:	Search for video files.

Thanks Alan.

**Comment by jjw24** [comment]
Hi there, apologies for the late reply. it is not implemented yet, but is definitely a good feature to add.

**Reply by AlanEdd** [reply]
So we could hotkey 'pic:' as the prefix and it would limit search to images, there are more like 'folder:' too. 
😃

**Reply by trACEroam** [reply, marked as answer]
i am using everything v1.5 instead of v1.4 with flow launcher...all your queries are work for me 😈
